frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -40,10 +40,18 @@ class OverrideForkPointDialog
 
   override fun createCenterPanel() =
       panel {
-        row(
-            getString(
-                "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name")) {
+        row {
+          label(
+              format(
+                  getString(
+                      "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name.1"),
+                  branch.name),
+              bold = false)
           label(branch.name, bold = true)
```

Comment:
Can you show some ss?

Reply:
yeah, screenshot would be useful

Reply:
This is how it looks like now:
![Screenshot from 2020-10-12 12-28-11](https://user-images.githubusercontent.com/56402877/95736911-296c1700-0c87-11eb-84ee-d4fa3501867d.png)
Unfortunately there are spaces between each `label()`

When I use the string like `<html>Select after which commit the range of commits belonging to <b>{0}</b> branch starts</html>` and code like `row(format(getString("action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name"), branch.name)) {}`, then it looks like that:
![Screenshot from 2020-10-12 12-31-15](https://user-images.githubusercontent.com/56402877/95737251-ab5c4000-0c87-11eb-898e-a95e02626716.png)
The last word is cut.

When we sightly stretch the window, then this word appears:
![Screenshot from 2020-10-12 12-31-24](https://user-images.githubusercontent.com/56402877/95737380-db0b4800-0c87-11eb-94ae-6ec70854002a.png)

First I think that this is an error with computation of rendered string width when we use a bold formatting, but even without `<b></b>` the last word is cut:
![Screenshot from 2020-10-12 12-32-57](https://user-images.githubusercontent.com/56402877/95737525-1c035c80-0c88-11eb-9cd6-4986d5aa97c2.png)

Finally this seems to be connected with `<html></html>` tags. When I use string like `Select after which commit the range of commits belonging to {0} branch starts` it seems to be rendered correctly:
![Screenshot from 2020-10-12 12-40-47](https://user-images.githubusercontent.com/56402877/95737664-5967ea00-0c88-11eb-9582-7b510dadb9ac.png)
But we lose a branch name emphasize that IMHO is a good (and important) thing here.

Reply:
Wooow that's weird indeed :hushed:
Let's ship as is then!

Reply:
Alright then, you could take a look and try to find some other way to display text like this (some substitute of label)

For reference:
https://jetbrains.org/intellij/sdk/docs/user_interface_components/kotlin_ui_dsl.html

Reply:
Great! Wrapping labels into single cell was enough.